### DISCUSSION TITLE
Not catching the webwidget_triggered webhook event

### DISCUSSION THREAD

**Discussion by ryukaizen** [chatwoot/chatwoot#10098] [Q&A]
Hello, I'm working on a Botpress - Chatwoot integration and want to filter in `webwidget_triggered` event as the use clicks on `Start Conversation` on that bubble web widget, so that Botpress could send the first message as a greeting.

The Botpress logs are showing every other events such as `message_created` etc. but not the webwidget one.

What are the factors that might be into play?

**Comment by sojan-official** [comment]
@ryukaizen  is this fixed ?

**Reply by ryukaizen** [reply]
@sojan-official what I noticed it sent `webwidget_triggered` only once

I had to clear browser cookies/cache, then tried opening the bubble widget from incognito mode, it fired that event only once per browser session.
